routing/pathfind_test.go

```diff
@@ -1443,6 +1443,87 @@ func TestRouteFailMinHTLC(t *testing.T) {
 	}
 }
 
+// TestRouteFailMaxHTLC tests that if we attempt to route an HTLC which is
+// larger than the advertised max HTLC of an edge, then path finding fails.
+func TestRouteFailMaxHTLC(t *testing.T) {
```

Comment:
This commit can be squashed with the one that introduces the pathfinding check.

Reply:
fixed